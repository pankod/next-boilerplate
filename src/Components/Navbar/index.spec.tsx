import * as React from "react";
import { shallow, mount } from "enzyme";

import { Navbar } from "@Components";

describe("Navbar", () => {
    it("should render without fail", () => {
        const wrapper = shallow(<Navbar />);

        expect(wrapper.find("div.navbar").exists()).toBe(true);
    });

    it("should match snapshot", () => {
        const wrapper = mount(<Navbar />);
        expect(wrapper).toMatchSnapshot();
    });
});
