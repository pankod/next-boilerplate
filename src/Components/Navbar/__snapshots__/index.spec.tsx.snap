// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match snapshot 1`] = `
<Navbar>
  <div
    className="navbar"
  >
    Navbar
  </div>
</Navbar>
`;
